### ISSUE TITLE
Docker: Database passwords with two exclamation points create invalid `config.yml`

### ISSUE THREAD

**Issue by axisofentropy** [Requarks/wiki#2293]
**Describe the bug**
The `wiki.js` container image doesn't seem to tolerate database passwords that have two exclamation points. It seems to generate a `config.yml` file that is not valid YAML.  Here's an example error message:

```
undeclared tag handle "!rE!" at line 8, column 25:
      pass: !rE!z-pzjvFKuVOu
                            ^
>>> Unable to read configuration file! Did you create the config.yml file?
```

**To Reproduce**
1. Start a Docker container using the `requarks/wiki` image and specifying the environment variable `DB_PASS` with value that includes two exclamation points. You may or may not need to specify other database connection environment variables.
2. Observe error in container logs.

**Expected behavior**
I expected the container to establish a database connection after constructing a valid `config.yml` file.
 
**Host Info (please complete the following information):**
 - OS: Google Kubernetes Engine, Rapid Release Channel, probably `1.17.8-gke.17`
 - Wiki.js version: Probably Docker tag `canary-2.5.90`
 - Database engine: Postgres 9 or 11, not sure

**Additional context**
Maybe just putting quotes around this value (and all others from environment variables!) within the YAML would fix this?

**Comment by Smankusors** [comment]
> Maybe just putting quotes around this value (and all others from environment variables!) within the YAML would fix this?

You... haven't tried this?

**Comment by NGPixel** [comment]
> You... haven't tried this?

He mentioned Docker env, which is set internally.

**Comment by ptts** [comment]
I'm pretty sure the issue is the _leading_ `!` and not the fact that it contains two `!`. Take a look at the [YAML specification about node tags](https://yaml.org/spec/1.2/spec.html#id2784064). Either quoting the password or not leading with an `!` should work.

**Comment by NGPixel** [comment]
Fixed by 29afc61abf6fbd6d09886ac7952b2e78c98633be
